### PR TITLE
[Ready for review] Updating references to Programming Historian

### LINKED ISSUE #3164: update link
### Summary

will need to update all reference to
https://programminghistorian.org/en/lessons/intro-to-bash that are present in the "Prerequisite" part of some chapters with the text [Experience with the command line](https://programminghistorian.org/en/lessons/intro-to-bash)

it should be:
```
 {ref}Experience with the command line<rr-overview-resources-commandline>
```

This is an easy fix for new people, should not be done before June 2023 to allow some time to merge the bookdash PRs (several chapters will be touched and we may otherwise get merge conflicts).
(this is a following issue after #310 was fixed)

### What needs to be done?

- [ ] check all chapter for prerequisite, or make a search over all file content
- [ ] modify the link at the different places

### Who can help?

_No response_

### Updates

- [ ] wait until June
- [ ] create branch
- [ ] Make changes
- [ ] Request reviews
- [ ] Address reviews
- [ ] Merge to main branch

### PR DESCRIPTION
**Pull request by kallewesterling**
### Summary

Because we are after June, I wanted to see if I could resolve #3164. This issue should fix #3164.

### List of changes proposed in this PR (pull-request)

- [X] update all reference to https://programminghistorian.org/en/lessons/intro-to-bash to follow the Sphinx standard reference system

### What should a reviewer concentrate their feedback on?

- [ ] Does everything render correctly on your end? (It does on mine)

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file: kallewesterling

### CODE REVIEW

**Comment by Arielle-Bennett** [comment]
@kallewesterling once we figure out why the deploy preview has gone wonky (not your fault!) I'll review this and get it moving 😄

**Comment by kallewesterling** [comment]
Hello @SaranjeetKaur -- I'm not sure what you've been reviewing, but my edits were not on the content of these pages, but (as you'll see in the #3164, only the way that the pages link to each other).

File: book/website/reproducible-research/ci.md
[the remark this answers is not in the document]
Comment (kallewesterling) [reply book/website/reproducible-research/ci.md:1]: Like I said in the comment below, I made no edits here -- this is already in the current book.

File: book/website/reproducible-research/make.md
[the remark this answers is not in the document]
Comment (kallewesterling) [reply book/website/reproducible-research/make.md:1]: Like I said in [the comment below](https://github.com/the-turing-way/the-turing-way/pull/3365#issuecomment-1808346811), I made no edits here -- this is already in the current book.

**Review by llewelld — APPROVED** [review]
This all looks good. The changes all rendered and worked correctly on my local build. Grepping the changes didn't throw up any other instances of "Experience with the command line" and the only place `programminghistorian` now appears is as a resource in the "Using the command line" sections as it should be. Manual review of the changes also looks excellent. Looks really great to me.

**Comment by SaranjeetKaur** [comment]
Hi @kallewesterling, I started with reviewing the whole chapter/section. Please ignore if it was not what was expected from this review.

**Comment by kallewesterling** [comment]
No worries @SaranjeetKaur -- just wanted to make sure that I provided some context to my edits here.

**Review by EKaroune — APPROVED** [review]
Looks all good to me!

**Pull request merged**